src/plugins/commander/commands/rc.js

```diff
@@ -0,0 +1,23 @@
+const Command = require('../structs/Command.js');
+
+class RCCommand extends Command {
+    constructor(bot) {
+        super(bot);
+        this.aliases = ['rc', 'rssbot', 'rctodiscord'];
+
+        this.shortdesc = `Information on setting up Recent Changes-to-Discord.`;
+        this.desc = `Tells you how to successfully set up a Recent Changes-to-Discord system.`;
```

Comment:
Mind changing the descriptions to say RC-to-Discord relay? It's probably a better description anyways, but mainly because the space annoys me